com.github.expresslrs.ExpressLRSConfigurator.appdata.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.expresslrs.ExpressLRSConfigurator</id>
```

Comment:
```suggestion
  <id>io.github.expresslrs.ExpressLRSConfigurator</id>
```
It should use "github.io" the namespace for pages rather than "github.com"


also since ` <url type="homepage">https://www.expresslrs.org/</url>` is in the appstream file, I'd assyme `org.expresslrs` would be used for the app-id.

Reply:
Change to homepage